src/renderer/components/SettingsSection/index.tsx

```diff
@@ -7,6 +7,30 @@ import CustomizationSettings from './Customization';
 import DownloadSettings from './Download';
 import settings from 'common/settings';
 import * as packageInfo from '../../../../package.json';
+import { Button, ButtonType } from '../Button';
+
+interface InstallButtonProps {
+    type?: ButtonType,
+    disabled?: boolean,
+    className?: string;
+    onClick?: () => void;
+}
+
+export const InstallButton: FC<InstallButtonProps> = ({
```

Comment:
I don't think this component is needed. Misnamed too

Reply:
I re-did it to make a ResetButton, should be OK